bank_integration/scripts/payment_entry.js

```
@@ -1,14 +1,15 @@
 // Copyright (c) 2018, Resilient Tech and contributors
 // For license information, please see license.txt
 
-frappe.ui.form.on('Payment Entry', {
-    setup: function(frm) {
-        frappe.realtime.on("eval_js", function(message){
+frappe.ui.form.on("Payment Entry", {
+    setup: function (frm) {
+        frappe.realtime.off("eval_js");
+        frappe.realtime.on("eval_js", function (message) {
             eval(message);
```

Biome (2.4.7)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)